.github/workflows/kotlin-test.yml

```
@@ -0,0 +1,47 @@
+name: Kotlin CI
+
+on:
+  push:
+    branches-ignore:
+      - master
+    paths: [ "app-android/**" ]
```

```suggestion
        paths:
      - 'app-android/**'
```
I have discovered square brackets in YAML are not arrays and are not recognized as such by GitHub Actions. This is the style used by the GitHub Actions' documentation.

---

both are supposed to be "sequences" in YAML, I wonder why "flow style" ones don't work

---

Yeah, it's weird. My only other hypothesis is the different quotation marks. YAML handles wildcards differently depending on quote style from memory. It still does not make much sense to me either way to be honest.